src/misc.jl

```diff
@@ -4,8 +4,8 @@
 #
 # See: https://en.wikipedia.org/wiki/Multivariate_gamma_function
 #
-function logmvgamma(p::Int, a::Float64)
-    res::Float64 = p * (p - 1.0) / 4.0 * log(pi)
+function logmvgamma(p::Int, a::Real)
+    res = p * (p - 1.0) / 4.0 * log(pi)
```

Comment:
I think I'm missing something.

1. Doesn't `log(pi)` on line 8 mean that res is always *at least* as big as `Float64`?
1. Where is the type instability again? If there's a cast, shouldn't it be
```julia
res::typeof(a) = p * (p - 1.0) / 4.0 * log(pi)
```